### ISSUE TITLE
Bozja/Eureka FATEs not clearing after completion

### ISSUE THREAD

**Issue by ZohiDev** [marzent/IINACT#59]
The Bozja/Eureka overlay module has an issue where none of the FATES are removed from the map after they're completed, quickly resulting in a map full of blinking text. Tested in Bozja & Zadnor

**Comment by marzent** [comment]
Yeah this is a known issue with the cactbot event sources regarding fates... There is *some* data getting lost for CE/fates specifically; I am not sure of the root cause yet.

**Comment by pistolwhip-dev** [comment]
Confirming that this is still happening.

In testing, ACT w/ default overlay plugin does not currently seem to have this issue.  FATEs are announced, % tracked, and cleared on completion as expected.

**Comment by thakyZ** [comment]
Just as a note for record keeping around this issue, there is a flag on the "Wolf" CE that is usually reserved for the Field Notes.   
This particular note I don't need, but as well as the note is not called "Nice".   
The only thing I can think of as a reason for the flag being labeled as "Nice" is because of another plugin that applies "Nice" to any damage floater that ends in 69.
![ffxiv_dx11_24-01-01_08 50 08-462](https://github.com/marzent/IINACT/assets/950594/a66341f7-85ad-4b91-b0a6-2f3793589516)
